app/transit_providers/be/stib/gtfs.py

```
@@ -1,89 +1,164 @@
+    for part in extra.split(","):
+        p = part.strip().lower()
+        if p:
+            out.add(p)
+    return out
+
+
+def _headers_for_file_url(file_url: str) -> dict:
+    """Only send APIM credentials to the mobility API host (not arbitrary URLs in listings)."""
+    try:
+        parsed = urlparse(file_url)
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)